dashboard/test/ui/features/support/connect.rb

```diff
@@ -58,6 +58,9 @@ def saucelabs_browser(test_run_name)
         http_client: http_client
       )
 
+      # Maximum time a single execute_script or execute_async_script command may take
+      browser.manage.timeouts.script_timeout = 30.seconds
```

Comment:
I'm not sure where the default was coming from, but without this line I was getting
```
Timeout expired waiting for async script
Command duration or timeout: 98 milliseconds
```